frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -72,22 +63,24 @@ function CriticalQuestions(props) {
   const [locationCount, setLocationCount] = useState(0);
 
   let nextPage;
-  const { story } = useSelector((state) => state.story);
+  const { story, status } = useSelector((state) => state.story);
+
   useEffect(() => {
     if (!story) dispatch(fetchStory());
     else setFormValues({ ...formValues, ...story });
   }, [dispatch, story]);
 
   const handleFormChange = (key) => (event) => {
-    const intFields = ["age"];
-    const nonValueFields = ["sicknessStart", "sicknessEnd"];
+    const intFields = [fields.AGE];
+    const nonValueFields = [fields.SICKNESS_START, fields.SICKNESS_END];
+    const keyName = key.name;
```

Comment:
Further evidence that this might benefit some naming refactor: the `key` is now a `field`, conceptually, and the `keyName` is the actual `key`